post-scan-actions/aws-python-promote-or-quarantine/README.md

```diff
@@ -1,322 +1,60 @@
-2. In a shell program, enter the following GNU Make command, using backslashes (`\`) to separate lines, as shown below:
-
-    ```bash
-    FUNCTION_NAME=<YOUR_FSS_FUNC_NAME> ROLE_ARN=<YOUR_FSS_ROLE_ARN> \
-    PROMOTE_BUCKET=<YOUR_PROMOTE_BUCKET> QUARANTINE_BUCKET=<YOUR_QUARANTINE_BUCKET> \
-    make create-function
-    ```
-- where:
-    - `<YOUR_FSS_FUNC_NAME>` is replaced with the name you want to give your Lambda function. Example: `FSS_Prom_Quar_Lambda`.
-    - `<YOUR_FSS_ROLE_ARN>` is replaced with the ARN of the role you previously created for the Lambda function. You can find the ARN in the AWS console under **Services > IAM > Roles** > your role > **Role ARN** field (at the top). Example: `arn:aws:iam::012345678901:role/FSS_Lambda_Role`.
-    - `<YOUR_PROMOTE_BUCKET>` is replaced with the name of your 'Promote bucket' as it appears in S3.
-    - `<YOUR_QUARANTINE_BUCKET>` is replaced with the name of your 'Quarantine bucket' as it appears in S3.
-</details>
-
-## Subscribe the Lambda to the SNS topic
-
-1. **Find the 'ScanResultTopic' SNS topic ARN**
+2. **Find the 'ScanResultTopic' SNS topic ARN**
     - In the AWS console, go to **Services > CloudFormation** > your all-in-one stack > **Resources** > your storage stack > **Resources**.
     - Scroll down to locate the  **ScanResultTopic** Logical ID.
     - Copy the **ScanResultTopic** ARN to a temporary location. Example: `arn:aws:sns:us-east-1:123445678901:FileStorageSecurity-All-In-One-Stack-StorageStack-1IDPU1PZ2W5RN-ScanResultTopic-N8DD2JH1GRKF`
-2. **Find the Lambda function ARN**
 
-    📌 The Lamdba function ARN is only required if you plan to use the AWS CLI (as opposed to the console) to subscribe the Lambda to the SNS topic.
-    - In the AWS console, go to **Services > Lambda**.
-    - Search for the Lambda function you created previously. Example: `FSS_Prom_Quar_Lambda`
-    - Click the Lambda function link.
-    - On the top-left, locate the **ARN**.
-    - Copy the ARN to a temporary location. Example: `arn:aws:lambda:us-east-1:123445678901:function:FSS_Prom_Quar_Lambda`
-3. **Subscribe the Lambda to the SNS topic**
+## Installation
 
-    <details>
-    <summary>Using the AWS console</summary>
+### From AWS Lambda Console
 
-    1. Go to **Services > Lambda**.
-    2. Search for the Lambda function you created previously. Example: `FSS_Prom_Quar_Lambda`
-    3. Click the link to your Lambda function to view its details.
-    4. Click **Add trigger** on the left.
-    5. From the **Trigger configuration** list, select **SNS**.
-    6. In the **SNS topic** field, enter the SNS topic ARN you found earlier.
-    7. Click **Add**. Your Lambda is now subscribed to the SNS topic.
+1. Visit [the app's page on the AWS Lambda Console](https://console.aws.amazon.com/lambda/home?region=us-east-1#/create/app?applicationId=arn:aws:serverlessrepo:us-east-1:415485722356:applications/cloudone-filestorage-plugin-action-promote-or-quarantine).
```

Comment:
Does region matter here? Do we need to ask customer to change region in the following step?

Reply:
The region of the console matters. I will remove that one.